src/components/outline/AppOutline.tsx

```diff
@@ -4,11 +4,34 @@ import { Footer, Service, apiUrl } from "@hex-labs/core";
 import Navigation from "./Navigation";
 import Display from "./Display";
 import { userData } from "../../definitions/DummyUserData";
+import axios from "axios";
+import { UserCardType } from "../../types/UserCard";
 
 const AppOutline: React.FC = () => {
   const [users, setUsers] = useState<any[]>([]);
 
-  // TODO: We need to get the hexathon users from the API
+  useEffect(() => {
+    const getUsers = async () => {
+      const hexUrl = apiUrl(Service.HEXATHONS, "/hexathon-users/647fee51768e521dc8ef88e0/users");
+      const hexData = await axios.get(hexUrl);
+      setUsers(hexData?.data?.hexathonUsers);
+    };
+    getUsers();
+  }, []);
```

Comment:
Change this to useAxios like i said [here](https://github.com/HackGT/match/pull/15#discussion_r1281392337), I guess I should've made it clear that the change was needed